examples/mock/mockoon-digitalregistries.json

```diff
@@ -1530,16 +1530,16 @@
             {
               "target": "params",
               "modifier": "ID",
-              "value": "",
-              "invert": true,
-              "operator": "null"
+              "value": "id231Q",
+              "invert": false,
+              "operator": "equals"
             },
             {
               "target": "params",
               "modifier": "ID",
-              "value": "ID1234",
+              "value": "[a-zA-Z]{2}\\d{3}[a-zA-Z0-9]{1}",
```

Comment:
This regex is very specific. I think we can keep it for now but latter on it might be required to introduce configuration in case of handling other systems.